src/pytest_mock_resources/fixture/database/relational/generic.py

```diff
@@ -154,6 +155,59 @@ def manage(self, session=None):
         finally:
             self.engine.dispose()
 
+    def manage_sync(self, session=None):
+        try:
+            self._run_actions()
+
+            if session:
+                if isinstance(session, sessionmaker):
+                    session_factory = session
+                else:
+                    session_factory = sessionmaker(bind=self.engine)
+
+                Session = scoped_session(session_factory)
+                session = Session(bind=self.engine)
+                yield session
+                session.close()
+            else:
+                yield self.engine
+        finally:
+            self.engine.dispose()
+
+    def manage_async(self, session=None):
+        try:
+            self._run_actions()
+
+            engine = self._get_async_engine()
+
+            if session:
+                if isinstance(session, sessionmaker):
+                    yield session
+                else:
+                    yield sessionmaker(
+                        engine, expire_on_commit=False, class_=AsyncSession
+                    )
+            else:
+                yield engine
+        finally:
+            self.engine.dispose()
+
+    def _get_async_engine(self, isolation_level=None):
```

Comment:
I'm not sure where there is both this, as well as the `container/async_/postgres.py` one. Should the container one disappear entirely, in favor of this one?

Reply:
Yes. I'll remove it in the next commit.